wremnants/datasets/datagroups.py

```diff
@@ -2036,8 +2036,9 @@ def histName(baseName, procName="", syst=""):
     @staticmethod
     def analysisLabel(filename):
         if filename not in Datagroups.mode_map:
-            raise ValueError(
+            logger.warning(
```

Comment:
Not sure if it ever comes out, but if you use a new histmaker which was not expected, now the code will just warn you, but then it continues, so what would happen ?